### PR TITLE
Docs: explicit note on updating Suite Guide

### PR DESCRIPTION
**Pull request by tsusanka**

### CODE REVIEW

File: docs/misc/guide.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ Guide is a feature that allows us to write content on various topics like basics
 
 See the [tech spec](../specs/guide.md) for details and rationale behind the implementation.
 
+## How to update
+
+Change `GITBOOK_REVISION` constant in `packages/suite-data/src/guide/constants.ts`. Example: [40855097](https://github.com/trezor/trezor-suite/commit/408550979cd58e78df297e30fb32e45935529a80). That's it.
+
```
Comment (tomasklim) [inline docs/misc/guide.md]: I would move it under `## Content` section because without reading that section, it is not clear what this section says.

Comment (goodhoko) [reply docs/misc/guide.md]: ^^ it's especially not clear what should I change it to.

Moving it *under* the `Content` section would help that. .)

Comment (tsusanka) [reply docs/misc/guide.md]: cc9cddc ?

---

I really should stop micro-managing and just let you guys working, sorry :innocent:.

File: docs/misc/guide.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ See the [tech spec](../specs/guide.md) for details and rationale behind the impl
...
+---
+
 The content is maintained in a [GitBook project](https://app.gitbook.com/@trezor/s/suite-product-guide/). (You'll need an account with appropriate privileges to access it.) GitBook mirrors the content to a [GitHub repository](https://github.com/trezor/trezor-suite-guide) from where it's fetched when Suite builds. The fetch happens in the `suite-data` package on `build:lib` yarn task. Once fetched the content is indexed and transformed to form usable in the Suite app. This form is then copied into the static directories of web, desktop and native builds.
 
 Indexing, transforming and copying of the content is handled by scripts in `packages/suite-data/src/guide`. These are configured to fetch a particular commit of the GitBook mirror by the `GB_REVISION` constant in `packages/suite-data/src/guide/constants.ts`. This constant must/can be changed accordingly to propagate changes from GitBook to suite builds. (One could set it to the `master` branch to always use the latest content version, however that is discouraged as we rather want to precisely control which version of the content will get included in each release.)
```
Comment (tomasklim) [inline docs/misc/guide.md]: It makes more sense in this order (at least for me)
```suggestion
The content is maintained in a [GitBook project](https://app.gitbook.com/@trezor/s/suite-product-guide/). (You'll need an account with appropriate privileges to access it.) GitBook mirrors the content to a [GitHub repository](https://github.com/trezor/trezor-suite-guide) from where it's fetched when Suite builds. The fetch happens in the `suite-data` package on `build:lib` yarn task. Once fetched the content is indexed and transformed to form usable in the Suite app. This form is then copied into the static directories of web, desktop and native builds.

Indexing, transforming and copying of the content is handled by scripts in `packages/suite-data/src/guide`. These are configured to fetch a particular commit of the GitBook mirror by the `GB_REVISION` constant in `packages/suite-data/src/guide/constants.ts`. This constant must/can be changed accordingly to propagate changes from GitBook to suite builds. (One could set it to the `master` branch to always use the latest content version, however that is discouraged as we rather want to precisely control which version of the content will get included in each release.)

### How to update

Change `GITBOOK_REVISION` constant in `packages/suite-data/src/guide/constants.ts` to the new revision. Example: [40855097](https://github.com/trezor/trezor-suite/commit/408550979cd58e78df297e30fb32e45935529a80). That's it.
```

Comment (tsusanka) [reply docs/misc/guide.md]: I don't really care. Feel free to commit this as a fixup commit.

**Review by tomasklim — APPROVED** [review]

**Pull request merged**